internal/app/app.go

```diff
@@ -0,0 +1,21 @@
+// Package app provides a function to start the application.
+package app
+
+import (
+	"net/http"
+
+	"github.com/Konstantsiy/image-converter/internal/auth"
+	"github.com/Konstantsiy/image-converter/internal/repository"
+	"github.com/Konstantsiy/image-converter/internal/server"
+	"github.com/gorilla/mux"
+)
+
+// Start starts the application server.
+func Start() error {
+	r := mux.NewRouter()
+	repo := repository.NewRepository()
+	tokenManager := auth.NewTokenManager("simple_private_key")
```

Comment:
`simple_private_key` is this just a stub? It will get the real value later for the config, right?

Reply:
> `simple_private_key` is this just a stub? It will get the real value later for the config, right?

Tis is just a stub for configs (the next PR)